honeybee_energy/lib/_loadconstructions.py

```diff
@@ -0,0 +1,35 @@
+"""Load all materials and constructions from the IDF libraries."""
+from honeybee_energy.construction import OpaqueConstruction, \
+    WindowConstruction
+
+import os
+import inspect
+
+
+# empty dictionaries to hold idf-loaded materials and constructions
+_idf_opaque_materials = {}
+_idf_window_materials = {}
+_idf_opaque_constructions = {}
+_idf_window_constructions = {}
+
+
+# load other materials and constructions from user-supplied files
+cur_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
what about using `__file__`?

```suggestion
cur_dir = os.path.dirname(__file__)
```

Reply:
Oh, wow, that's so much cleaner.  I see that it also works well for getting the absolute path in IronPython. Ok, I will fix it here and I also have another place to apply this change: https://github.com/ladybug-tools/ladybug-comfort/pull/21